tempesta_fw/t/functional/leaks/test_leak.py

```diff
@@ -0,0 +1,110 @@
+"""
+Testing for memory leaks
+"""
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2017 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+import os.path
+import subprocess
+
+import psutil
```

Comment:
Pylint2: `E: 12, 0: Unable to import 'psutil' (import-error)`. It's a new library required for the tests. Update to Readme is required.

Reply:
fixes